build/build-info

```diff
@@ -20,18 +20,21 @@
 # Arguments:
 #   version  - The current version of the project
 #   git_path - The path to the repository
+#   library_path - The path to the libcudf library
 set -e
 
 echo_build_properties() {
   version=$1
   git_path=$2
-  shift 2
+  library_path=$3
+  shift 3
   echo version=$version
   echo user=$USER
   echo revision=$(cd "$git_path" && git rev-parse HEAD)
   echo branch=$(cd "$git_path" && git rev-parse --abbrev-ref HEAD)
   echo date=$(date -u +%Y-%m-%dT%H:%M:%SZ)
   echo url=$(cd "$git_path" && git config --get remote.origin.url)
+  echo gpu_architectures=$(cd "$git_path" && cuobjdump "$library_path" 2>/dev/null | grep 'arch = ' | awk -F_ '{print $2}' | sort -n -u | tr '\n' ';')
```

Comment:
nit: not a git op and no need to cd?
```suggestion
  echo gpu_architectures=$(cuobjdump "$library_path" 2>/dev/null | grep 'arch = ' | awk -F_ '{print $2}' | sort -n -u | tr '\n' ';')
```

Reply:
It kept it as safety but the build paths are absolute. Refactored as suggested.